nexus/src/app/background/tasks/blueprint_planner.rs

```diff
@@ -24,7 +25,7 @@ use tokio::sync::watch::{self, Receiver, Sender};
 /// Background task that runs the update planner.
 pub struct BlueprintPlanner {
     datastore: Arc<DataStore>,
-    disabled: bool,
+    rx_chicken_switches: Receiver<Option<ReconfiguratorChickenSwitches>>,
```

Comment:
I understand why this is an `Option`, but does it need to be? Presumably for every possible chicken switch, we have _some_ default value (i.e., the thing we'd choose if this is `None`). Could we populate the channel with a `ReconfiguratorChickenSwitches` with those values set to avoid having to deal with `None` at all in this task?

Oh, as I write that I guess we don't have a default for `version`. How gross would it be to fill in a `version: 0` for this "made up default" set of switches?

Reply:
We'd also need a default for the timestamp, which is even weirder IMO. However, I agree that it would be better to use a default. The primary reason is that we wouldn't have to encode the defaults in code here at different points, and can just put them in a default impl.  I'll make this change.

Reply:
Done in [03cda10](https://github.com/oxidecomputer/omicron/pull/8462/commits/03cda10b498566b5b7ac7910eb34f32c4139bde9)